kalite/distributed/features/steps/superuser_create.py

```diff
@@ -46,6 +52,8 @@ def step_impl(context):
 
 @then("the modal won't dismiss")
 def step_impl(context):
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(2)
```

Comment:
They can be shared, it's defined in the environment functions (I made a comment about it below).